CRM/Core/SelectValues.php

```diff
@@ -1177,4 +1177,17 @@ public static function optionValueFields() {
     ];
   }
 
+  /**
+   * Limit-to options for schedule reminders.
+   *
+   * @return array
+   */
+  public static function getLimitToValues() {
+    return [
+      1 => ts('Neither'),
```

Comment:
:clap:  Glad that "-neither-"  is being replaced by "Neither" much better for a translatable string.